src/plot_api/plot_api.js

```diff
@@ -3027,10 +3051,14 @@ function makePlotFramework(gd) {
     // TODO: sort out all the ordering so we don't have to
     // explicitly delete anything
     fullLayout._glcontainer = fullLayout._paperdiv.selectAll('.gl-container')
-        .data([0]);
+        .data([{}]);
```

Comment:
why `[0]` to `[{}]` ?

Reply:
We reuse this object later in parcoord as container for data.